src/content/reference/cli.mdx

```diff
@@ -604,6 +604,22 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 | :-------- | :----: | :------------------------------------------------------------------------------------- |
 | _--label_ | (list) | tag and organize preview environments using labels (multiple _--label_ flags accepted) |
 
+#### sleep
+
+Sleeps an okteto preview environment. You need to provide that preview name as an argument to the command.
```

Comment:
The "that preview name" sounds a bit odd to me, what do you think of something more concise like:

```suggestion
Sleeps a given okteto preview environment
```

or 

```suggestion
Puts the specified okteto preview environment into sleep mode
```